gradle.properties

```diff
@@ -1,10 +1,9 @@
-enablePublishing=false
 fiatVersion=1.26.0
 korkVersion=7.76.3
-keikoVersion=3.9.0
```

Comment:
Ooh, spinnaker/orca#3897! I hadn't seen that!

Is that what you're working on, having kayenta use the new keiko-integrated orca?

Reply:
yeah, that is the first trip to the yak barbershop ..